### PR TITLE
Re-edit of PGD - third set of edits

### PR DESCRIPTION
**Pull request by ebgitelman**

### CODE REVIEW

File: product_docs/docs/pgd/5/deploy-config/deploy-tpa/deploying/02-deploying.mdx [hunk 1]
```diff
@@ -25,6 +25,8 @@ At the end of the provisioning stage, you will have the required number of insta
 ## Deploy
 The `tpaexec deploy` command installs and configures Postgres and other software on the provisioned servers. TPA can create the servers, but it doesn't matter who created them so long as SSH and sudo access are available. This includes setting up replication, backups, and so on.
```
Comment (ebgitelman) [inline product_docs/docs/pgd/5/deploy-config/deploy-tpa/deploying/02-deploying.mdx:26]: I couldn't trace what "This" means in the last sentence. Can you clarify?

File: product_docs/docs/pgd/5/deploy-config/deploy-tpa/deploying/02-deploying.mdx [hunk 2]
```diff
@@ -25,6 +25,8 @@ At the end of the provisioning stage, you will have the required number of insta
...
 
+<!-- What is "this">
```
Comment (ebgitelman) [inline product_docs/docs/pgd/5/deploy-config/deploy-tpa/deploying/02-deploying.mdx:28]:
```suggestion
```

File: product_docs/docs/pgd/5/durability/camo.mdx [hunk 3]
```diff
@@ -79,7 +79,9 @@ This doesn't allow COMMIT status to be retrieved, but it does let you choose ava
...
+The switch from CAMO-protected to asynchronous mode is only ever triggered by an actual CAMO transaction. This is true either because the commit exceeds the `timeout` value of `TO ASYNC` or, in case the CAMO partner is already known, disconnected at the time of commit. This switch is independent of the estimated catchup interval. If the CAMO pair is configured to require the current node to be the write lead of a group as configured through the `enable_proxy_routing` node group option. See [Commit scopes](commit-scopes) for syntax. This can prevent a split brain situation due to an isolated node from switching to asynchronous mode. If `enable_proxy_routing` isn't set for the CAMO group, the origin node switches to asynchronous mode immediately.
```
Comment (ebgitelman) [inline product_docs/docs/pgd/5/durability/camo.mdx:82]: This is a fragment: 
If the CAMO pair is configured to require the current node to be the write lead of a group as configured through the `enable_proxy_routing` node group option.
Can you figure out what was meant? 

Also, I was again not able to trace what "This" meant in the second-to-last sentence.

File: product_docs/docs/pgd/5/durability/camo.mdx [hunk 4]
```diff
@@ -79,7 +79,9 @@ This doesn't allow COMMIT status to be retrieved, but it does let you choose ava
...
+
+<!-- Sentence fragment, this what? >
```
Comment (ebgitelman) [inline product_docs/docs/pgd/5/durability/camo.mdx:84]:
```suggestion
```

File: product_docs/docs/pgd/5/durability/camo.mdx [hunk 5]
```diff
@@ -131,59 +133,61 @@ CAMO relies on a retry loop and specific error handling on the client side. Ther
...
+If it was a bad connection, then you can check on the CAMO partner node to see if the transaction made it there.
 
 ```
         conn = PQconnectdb(partner_dsn);
         if (!connectionEstablished())
             panic();
 ```
 
-If we can't connect to the partner node, there's not a lot we can do. So we should panic, or take similar actions.
+If you can't connect to the partner node, there's not a lot you can do. So we should panic, or take similar actions.
```
Comment (ebgitelman) [inline product_docs/docs/pgd/5/durability/camo.mdx:186]: I don't think "panic" is a technical term? I needed to change this to second person, but I wasn't sure if this was humor (which we shouldn't use) or what.

File: product_docs/docs/pgd/5/durability/camo.mdx [hunk 6]
```diff
@@ -131,59 +133,61 @@ CAMO relies on a retry loop and specific error handling on the client side. Ther
...
+
+<!-- surely not >
```
Comment (ebgitelman) [inline product_docs/docs/pgd/5/durability/camo.mdx:188]:
```suggestion
```

File: product_docs/docs/pgd/5/deploy-config/deploy-tpa/index.mdx [hunk 7]
```diff
@@ -17,13 +17,13 @@ both self-hosted and in the cloud (with AWS EC2).
 
     If you want to run on your own Linux systems or VMs, you can use also use TPA to [deploy EDB Postgres Distributed directly to your own Linux hosts](/pgd/latest/quickstart/quick_start_linux)
```
Comment (ebgitelman) [inline product_docs/docs/pgd/5/deploy-config/deploy-tpa/index.mdx:18]: I believe this should be a relative link.

**Comment by ebgitelman** [comment]
@djw-m I have a few queries on this one. Thank you!

File: product_docs/docs/pgd/5/durability/camo.mdx (hunk 5 above)
Comment (djw-m) [reply product_docs/docs/pgd/5/durability/camo.mdx:185]: It is - panic means to dump core and exit with an error. for example in Go language it is a standard function to panic when an error condition can't be resolved.

File: product_docs/docs/pgd/5/deploy-config/deploy-tpa/index.mdx (hunk 7 above)
Comment (djw-m) [reply product_docs/docs/pgd/5/deploy-config/deploy-tpa/index.mdx:18]: Yes

File: product_docs/docs/pgd/5/durability/camo.mdx (hunk 5 above)
Comment (ebgitelman) [reply product_docs/docs/pgd/5/durability/camo.mdx:185]:
```suggestion
If you can't connect to the partner node, there's not a lot you can do. In this case, panic, or take similar actions.
```

File: product_docs/docs/pgd/5/deploy-config/deploy-tpa/index.mdx (hunk 7 above)
Comment (ebgitelman) [reply product_docs/docs/pgd/5/deploy-config/deploy-tpa/index.mdx:18]: Do you want me to fix these or enter a bug?

File: product_docs/docs/pgd/5/durability/camo.mdx [hunk 8]
```diff
@@ -256,25 +258,25 @@ forced to roll back.
...
 ## CAMO limitations
 
-CAMO limitations are covered on the [Durability limitations](limitations/#camo) page.
+CAMO limitations are covered in [Durability limitations](limitations/#camo).
 
 ## Performance implications
 
-CAMO extends the Postgres replication protocol by adding a message roundtrip at commit. Applications have a higher
-commit latency than with asynchronous replication, mostly determined by the roundtrip time between involved nodes. Increasing the number of concurrent sessions can help to increase parallelism to obtain reasonable transaction throughput.
+CAMO extends the Postgres replication protocol by adding a message round trip at commit. Applications have a higher
```
Comment (djw-m) [inline product_docs/docs/pgd/5/durability/camo.mdx:275]:
```suggestion
CAMO extends the Postgres replication protocol by adding a message roundtrip at commit. Applications have a higher
```

**Review by djw-m — APPROVED** [review]
LGTM - will need to return to the broken text in the CAMO switch section,

**Pull request merged**